presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchConnectorConfig.java

```diff
@@ -54,6 +65,15 @@ public void testExplicitPropertyMappings()
                 .put("elasticsearch.request-timeout", "1s")
                 .put("elasticsearch.max-request-retries", "3")
                 .put("elasticsearch.max-request-retry-time", "5s")
+                .put("searchguard.ssl-transport-certificate_format", "PEM")
```

Comment:
Use `-` for `certificate-format`

We normally use dots to group related functionality in configs. I would name these like

* `searchguard.ssl-transport.certificate-format`
* `searchguard.ssl-transport.pemcert-filepath`

Or shorten to 

* `searchguard.ssl.certificate-format`

Since `ssl-transport` seems redundant